Makefile.Common

```diff
@@ -162,9 +163,13 @@ fmt: $(GOIMPORTS)
 	$(GOIMPORTS) -w  -local github.com/open-telemetry/opentelemetry-collector-contrib ./
 
 .PHONY: lint
-lint: $(LINT) checklicense misspell
+lint: $(LINT) checklicense misspell govulncheck
 	$(LINT) run --allow-parallel-runners --build-tags integration
 
+.PHONY: govulncheck
+govulncheck:
```

Comment:
```suggestion
govulncheck: $(GOVULNCHECK)
```

Reply:
Thank you :) But why do you suggest that? My reasoning was this is a command invoke so I placed that in its own indented line below, not as a dependency target in the same line.

As a side note, I bet `$(GOVULNCHECK) ./...` is the correct call.

Reply:
Adding that dependency will ensure the `govulncheck` binary is installed before running the make target `govulncheck`

Reply:
silly me, you're right, thank you so much for help! :)